test/lib/cuentas.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../helpers');
+const lab = exports.lab = Lab.script();
+const cuentas = require('../../lib/cuentas');
+
+
+describe('cuentas', () => {
+  describe('caulcular total', () => {
+    it('should calculate total when there is no tip on itbms', () => {
+      const results = cuentas(5, .10, 0);
```

Comment:
Aunque 0 también es valido, como esto es boolena, for continuity's sake, usa `false`
```suggestion
      const results = cuentas(5, .10, false);
```